_layouts/default.html

```diff
@@ -97,7 +100,7 @@ <h1 class="page_banner__title">{{ page.title }}</h1>
     </div>
 </footer>
 
-<script src="/js/jquery.min.js"></script>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```

Comment:
https is safer, now google cdn, cdnjs and jsdelivr provides https prefix url by default, what do you think?